plebnet/vpn/mullvad.py

```diff
@@ -63,24 +120,108 @@ def pay(self, price, bitcoin_address):
 			print('Insufficient balance, transaction cancelled')
 			return False
 
+	#Check if VPN is active
+	def checkVPN(self, setup=False):
+		res = requests.get("https://am.i.mullvad.net/json")
+		print(res.json())
+		#Check if ip country is Sweden
+		if res.json()['country'] == "Sweden":
+			print("VPN is active!")
+		else:
+			if setup:
+				print("Error: VPN was not installed!")
+			else:
+				self.setupVPN() 
+
 	#Setup the VPN
-	def setupVPN():
+	def setupVPN(self):
 		print("Time to setup the vpn!")
+		self.downloadFiles()
 		#TODO: Setup VPN using openVPN and mullvad settings
+		#Update and install openvpn
+		#os.popen("sudo apt-get update && apt-get upgrade")
+		test = os.popen("sudo apt-get install openvpn").read()
+		print(test)
+		#copy files to openvpn folder
+		test = os.popen("sudo cp -a ./config-files/. /etc/openvpn/").read()
+		print(test)
+		os.chdir("/etc/openvpn/")
+		#start openvpn service
+		#test = os.popen("sudo nohup openvpn --config ./mullvad_se-sto.conf > /dev/null &").read()
+		test = os.popen("sudo service openvpn start").read()
```

Comment:
Are yoy sure we need the service command? It didn't work for me.

Reply:
Hmm... It did work for me now. If that does not work, then you can comment that line and uncomment the line above. I used this so that it is easy to start/stop the vpn. With the other line, you have to find and then kill the process.